Analyzer/src/Analyze0Lep.cc

```diff
@@ -487,6 +487,8 @@ void Analyze0Lep::Loop(NTupleReader& tr, double weight, int maxevents, bool isQu
 
 void Analyze0Lep::WriteHistos(TFile* outfile)
 {
+    outfile->cd();
```

Comment:
Does this line actually do anything? I'm just curious because i'm not sure what this command does since there is only one out file.

Reply:
Yes, without it, none of the histograms will write out to your outfile. This is because ROOT's memory allocation ends up on the file last opened, and when we put in the ROOT files for the efficiencies (Pileup....root and allInOne...root), ROOT will try to write the histograms to those read only files and it will fail. 

If you're curious, you can run the analyzer without that line, and see what happens.

Reply:
Oh I see thanks.